packages/components/src/components/input-field/input-field.tsx

```diff
@@ -89,7 +90,7 @@ const InputField = ({
     id,
     inline_prefix,
     is_autocomplete_disabled,
-    is_disabled,
+    is_disabled = false,
```

Comment:
```suggestion
    is_disabled,
```
@kate-deriv default value of false is unnecessary because is_disabled is used as boolean value in the code anyway: !!is_disabled :)